Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/AssetFolderThumbnailView.cpp

```diff
@@ -428,6 +428,13 @@ namespace AzQtComponents
             {
                 emit afterRename(value);
             });
+
+        // Enable drag/drop.
+        setDragEnabled(true);
+        setAcceptDrops(true);
+        setDragDropMode(QAbstractItemView::DragDrop);
+        setDropIndicatorShown(true);
+        setDragDropOverwriteMode(true);
```

Comment:
Would you mind deleting these 5 calls? They are duplicated from L417-L421 of this file